src/Workspaces/Core/Portable/Editing/DeclarationModifiers.cs

```diff
@@ -68,7 +71,8 @@ public static DeclarationModifiers From(ISymbol symbol)
                 isOverride: symbol.IsOverride,
                 isSealed: symbol.IsSealed,
                 isConst: field != null && field.IsConst,
-                isUnsafe: symbol.IsUnsafe());
+                isUnsafe: symbol.IsUnsafe(),
+                isVolatile: field != null && field.IsVolatile);
```

Comment:
Is `symbol.IsExtern` needed here?

Reply:
nope.  fields can't be extern.

Reply:
Looks like this method can be called on a method symbol as well, right?
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/Editing/DeclarationModifiers.cs,b81ef0f65307b91e,references

Reply:
oh you're right.  Thanks for catching htat!